src/Imath/ImathMatrixAlgo.h

```diff
@@ -779,18 +779,17 @@ extractQuat (const Matrix44<T>& mat)
 {
     Matrix44<T> rot;
 
-    T       tr, s;
+    T       s;
     T       q[4];
     int     i, j, k;
     Quat<T> quat;
 
     int nxt[3] = {1, 2, 0};
-    tr         = mat[0][0] + mat[1][1] + mat[2][2];
```

Comment:
This is not correct. mat.trace() includes mat[3][3], but tr in the old formulation does not. Thus the test failure.

Reply:
Oh I see, sorry about that :( I'll revert